urdf2webots/writeProto.py

```diff
@@ -156,17 +156,26 @@ def URDFLink(proto, link, level, parentList, childList, linkList, jointList, sen
             proto.write((level + 2) * indent + 'density -1\n')
             proto.write((level + 2) * indent + 'mass %lf\n' % link.inertia.mass)
             if link.inertia.ixx > 0.0 and link.inertia.iyy > 0.0 and link.inertia.izz > 0.0:
+                i = link.inertia
+                inertiaMatrix = [i.ixx, i.ixy, i.ixz, 0, i.iyy, i.iyz, 0, 0, i.izz]
+                if link.inertia.rotation[-1] != 0.0: 
+                    rotationMatrix = matrixFromRotation(link.inertia.rotation)
+                    I = np.array(inertiaMatrix).reshape(3, 3)
+                    R = np.array(rotationMatrix).reshape(3, 3)
+                    R_t = np.transpose(R)
+                    # calculate the rotate inertiaMatrix with R_t + I * R. For reference, check the link below
+                    # https://www.euclideanspace.com/physics/dynamics/inertia/rotation/index.htm
+                    inertiaMatrix = np.dot(np.dot(R_t,I), R).reshape(9) 
                 proto.write((level + 2) * indent + 'centerOfMass [ %lf %lf %lf ]\n' % (link.inertia.position[0],
                                                                                        link.inertia.position[1],
                                                                                        link.inertia.position[2]))
```

Comment:
This part should probably be removed from this if block and moved in a new block:
```Python
if link.inertia.position[0] != 0 or link.inertia.position[1] != 0 or link.inertia.position[2] != 0:
    ...
```

Reply:
can you elaborate? Do you mean the centerOfMass needs a condition?

Reply:
Yes sure, what I mean is that currently, the center of mass is exported only if:
```Python
if link.inertia.ixx > 0.0 and link.inertia.iyy > 0.0 and link.inertia.izz > 0.0:
```
But this is not related to the center of mass, but rather to the condition.
The center of mass part should therefore be moved outside of this if.
But to avoid exporting default value (`[0, 0, 0]`), we should add a condition to test that it is not the default value already.

Reply:
put it outside of this condition